verification/initializeContext.go

```diff
@@ -67,3 +67,27 @@ func testInitContext(d TestDPEInstance, client DPEClient, t *testing.T, simulati
 		}
 	}
 }
+
+func getContextHandle(d TestDPEInstance, c DPEClient, t *testing.T, simulation bool) *ContextHandle {
```

Comment:
nit: rename to `getInitialContextHandle`? Also would be good to have a function level comment explaining what this function is for.

Reply:
Yes renamed and added a description.